app/Livewire/Upgrade.php

```
@@ -41,4 +51,71 @@ public function upgrade()
+        }
+
+        $parts = explode('|', $content);
+        if (count($parts) < 3) {
+            return ['status' => 'none'];
+        }
+
+        [$step, $message, $timestamp] = $parts;
+
+        // Check if status is stale (older than 10 minutes)
+        try {
+            $statusTime = new \DateTime($timestamp);
```

PHPMD (2.15.0)

Missing class import via use statement (line '93', column '31'). (undefined)

(MissingImport)